baselines/diabetic_retinopathy_detection/README.md

```diff
@@ -34,6 +34,87 @@ pip install "git+https://github.com/google-research/robustness_metrics.git#egg=r
 pip install 'git+https://github.com/google/edward2.git'
 ```
 
+## Data Installation
+
+Because the data is distributed through Kaggle, we need to take a manual route to downloading.
+
+1. Download from Kaggle: https://www.kaggle.com/c/diabetic-retinopathy-detection
+
+2. Extract everything to ``$DATA_DIR/downloads/manual``; your directory should look like
+
+``sample/  sampleSubmission.csv  test/  train/  trainLabels.csv``
+
+3. Confirm successful download of files
+
+You should have 35,126 training images and 53,576 test images, which should be located in manual/train and manual/test.
```

Comment:
these lines need indent to format the list correctly. I'll merge now though since we're working in different time zones